### ISSUE TITLE
unstable Q1-DGP0 element?

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#2637]
We currently allow running with ``stokes_velocity_degree=1`` with discontinuous pressures, see https://github.com/geodynamics/aspect/blob/master/source/simulator/parameters.cc#L1384 but this element is not inf-sup stable (see Ern/Guermond p 186 for example).
I am confused that this works! Am I missing something? I don't think we have pressure stabilization implemented yet, right?

It is currently only used in tests/initial_condition_S20RTS_dgp.prm (but it is commented out in several other .prm files like tests/annulus.prm, benchmarks/hollow_sphere/hollow_sphere.prm, ...).

**Comment by cedrict** [comment]
yes, I have used this element in the annulus and hollow sphere benchmarks to compare ASPECT's results with ELEFANT results. Please don't switch this off forever :)

**Comment by tjhei** [comment]
> yes, I have used this element

but it is unstable and the solutions are therefore useless?

**Comment by bangerth** [comment]
Whether or not you actually do get instabilities, and how large they are, depends on the particular setup. It's not like "unstable" means that the solution *must* be useless -- it *may* be useless, however.

**Comment by cedrict** [comment]
If you say that the solutions are useless then you can throw away 20-30 years of geodynamics papers with CITCOM and other codes :)
More seriously Gresho and Sani make a case for the element in their book. Pressure checkerboard modes do not always occur, and the velocity is fine. ( see for instance my paper on the hollow sphere benchmark https://www.solid-earth.net/8/1181/2017/se-8-1181-2017.pdf ). You may wish to remove the option from the input file with an assertthrow but please leave the door open for ppl like me who wish to test ASPECT against their own code.

**Comment by tjhei** [comment]
> not like "unstable" means that the solution must be useless -- it may be useless, however.

So do you propose to leave everything as is?

**Comment by bangerth** [comment]
I suspect the number of people playing with this parameter is small. We can document that choosing 1 is probably not a good idea, and why, but that leaves the door open for @cedrict .
